src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  let fullAmount = 0;
+
+  for (let i = 1; i <= days; i++) {
+    fullAmount += rentDay;
+  }
+
+  if (days >= 7) {
+    fullAmount = fullAmount - 50;
+
+    return fullAmount;
+  }
+
+  if (days >= 3) {
+    fullAmount = fullAmount - 20;
+
+    return fullAmount;
```

Comment:
Similar to the previous comment, avoid using literals for the discount value. Use a constant like 'const SHORT_TERM_DISCOUNT = 20;' to make the code more readable and maintainable.